### PR TITLE
Dev 36 - implemented 404 page

### PR DESCRIPTION
**Pull request by KovDimaY**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure all the tests are still passing.
* [x] Make sure the test coverage is still 100%.
* [x] Make sure your source branch is correct.
* [x] Make sure your destination branch is **devel**.
* [x] Make sure the name of the PR has a correct DEV-# identification prefix.
* [x] Make sure you reviewed all your changes and they look correct.
* [x] Make sure all the commits have a correct format. 
* [x] Make sure you refer related issues for you PR. 
* [x] Make sure you add **KovDimaY** as a reviewer and assignee of your PR.

Take a look at the Issue #61

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "react-router-dom": "4.3.1",
     "react-scripts": "2.1.1",
     "react-sweet-progress": "1.1.2",
+    "react-tilt": "^0.1.4",
```
Comment (KovDimaY) [inline client/package.json]: Better to fix the version

File: client/src/components/Pages/404/styles.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  .not-found-container .image {
+    width: 200px;
+    height: 129px;
+  }
+
+  .not-found-container .image.text {
+    transform: translateZ(15px);
+  }
+
+  .not-found-container .image.text {
```
Comment (KovDimaY) [inline client/src/components/Pages/404/styles.css]: These styles are duplicated with the previous media query.

File: client/package.json (hunk 1 above)
Comment (KovDimaY) [reply client/package.json]: fixed

File: client/src/components/Pages/404/styles.css (hunk 2 above)
Comment (KovDimaY) [reply client/src/components/Pages/404/styles.css]: fixed

**Comment by TetianaKlymchuk** [comment]
Looks great! Good job! I do approval.

**Pull request merged**